### PR TITLE
Split pkg repos

### PR DESCRIPTION
**Pull request by kmoore134**

### CODE REVIEW

**Comment by MiwiDots** [comment]
LGTM. Thanks!

File: release/release-trueos.sh [hunk 1]
```diff
@@ -76,54 +77,54 @@ setup_poudriere_conf()
...
-	echo "GIT_URL=${GH_PORTS}" >> ${POUDRIERE_ETCDIR}/poudriere.conf
-	echo "USE_TMPFS=data" >> ${POUDRIERE_ETCDIR}/poudriere.conf
-	echo "BASEFS=$POUDRIERE_BASEFS" >> ${POUDRIERE_ETCDIR}/poudriere.conf
-	echo "ATOMIC_PACKAGE_REPOSITORY=no" >> ${POUDRIERE_ETCDIR}/poudriere.conf
-	echo "PKG_REPO_FROM_HOST=yes" >> ${POUDRIERE_ETCDIR}/poudriere.conf
+	echo "GIT_URL=${GH_PORTS}" >> ${_pdconf}
+	echo "USE_TMPFS=data" >> ${_pdconf}
+	echo "BASEFS=$POUDRIERE_BASEFS" >> ${_pdconf}
+	echo "ATOMIC_PACKAGE_REPOSITORY=no" >> ${_pdconf}
+	echo "PKG_REPO_FROM_HOST=yes" >> ${_pdconf}
```
Comment (MiwiDots) [inline release/release-trueos.sh]: We might want to add ALLOW_MAKE_JOBS_PACKAGES="pkg py* gcc* webkit* llvm* clang* perl* firefox* ruby* cmake* binutils p5* php* rust*" to speed up the build.

Comment (kmoore134) [reply release/release-trueos.sh]: Agreed, but should that be here, or in /etc/poudriere.conf.release so you can customize it on an individual build node?

Comment (kmoore134) [reply release/release-trueos.sh]: Ok, since that can be overridden I've gone ahead and added it.

**Pull request merged**